src/pages/myAccount/MyAccount.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Link, Outlet } from "react-router-dom";
+
+const MyAccount = () => {
+    return (
+        <div style={{ display: "flex", flexDirection: "column" }}>
+            MyAccount
+            <Link to="/my-account">PersonalInfo</Link>
+            <Link to="/my-account/my-posts">My Posts</Link>
+            <Link to="/my-account/my-drafts">My Drafts</Link>
+            <Link to="/my-account/my-bookmarks">My Bookmarks</Link>
```

Comment:
I thought that I will be a tab section instead of links

Reply:
Remember that this is just an initial structure for the application. Not final implementation

Reply:
Noted 🙂